server/src/main/resources/org/elasticsearch/common/reference-docs-links.json

```diff
@@ -43,5 +43,6 @@
   "MAX_SHARDS_PER_NODE": "size-your-shards.html#troubleshooting-max-shards-open",
   "FLOOD_STAGE_WATERMARK": "fix-watermark-errors.html",
   "X_OPAQUE_ID": "api-conventions.html#x-opaque-id",
-  "FORMING_SINGLE_NODE_CLUSTERS": "modules-discovery-bootstrap-cluster.html#modules-discovery-bootstrap-cluster-joining"
+  "FORMING_SINGLE_NODE_CLUSTERS": "modules-discovery-bootstrap-cluster.html#modules-discovery-bootstrap-cluster-joining",
+  "ALLOCATION_EXPLAIN_NO_COPIES": "cluster-allocation-explain.html#_no_valid_shard_copy"
```

Comment:
Please add a fixed `[[anchor-name]]` to the docs rather than using the `#_auto-generated` one that might change inadvertently.

Reply:
@matthewabbott, elaborating ... 

On [this doc line](https://github.com/elastic/elasticsearch/edit/main/docs/reference/cluster/allocation-explain.asciidoc#L210), you'll want to prefix a line like [this example](https://github.com/elastic/elasticsearch/blob/main/docs/reference/redirects.asciidoc?plain=1#L1920-L1922). So we will be editing the `*.asciidoc` in this PR after all. Afterwards in the above, what you had set inside the square brackets like `[[explain-no-valid-shard-copy]]` will become what goes at the end of the URL `cluster-allocation-explain.html#explain-no-valid-shard-copy`.